docs/docs/guide/debugging_tables/tables.tsx

```diff
@@ -0,0 +1,262 @@
+            <td style={{ textAlign: 'center' }}>✅ ¹</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td rowSpan={2}>
+              <a href="#chrome-devtools">Chrome DevTools</a>
+            </td>
+            <td style={{ textAlign: 'center' }}>Android</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+          </tr>
+          <tr>
+            <td style={{ textAlign: 'center' }}>iOS</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td rowSpan={2}>
+              <a href="#flipper-hermes-debugger">Flipper (Hermes debugger)</a>
+            </td>
+            <td style={{ textAlign: 'center' }}>Android</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+          </tr>
+          <tr>
+            <td style={{ textAlign: 'center' }}>iOS</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>✅ ²</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td rowSpan={2}>
+              <a href="#safari-devtools">Safari DevTools</a>
+            </td>
+            <td style={{ textAlign: 'center' }}>Android</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td style={{ textAlign: 'center' }}>iOS</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+          <tr>
+            <td rowSpan={2}>
+              <a href="#react-developer-tools">React Developer Tools</a>
+            </td>
+            <td style={{ textAlign: 'center' }}>Android</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+          </tr>
+          <tr>
+            <td style={{ textAlign: 'center' }}>iOS</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>✅</td>
+            <td style={{ textAlign: 'center' }}>N/A</td>
+          </tr>
+        </tbody>
+      </table>
+      ¹ - Works, but uses web implementations of function and runs everything on
+      the JS thread.
+      <br />² - Works, but only the JS context can be debugged.
+      <h4 style={{ marginBottom: 0 }}>Legend:</h4>
```

Comment:
Maybe let's add some bottom margin.
<img width="325" alt="Zrzut ekranu 2022-08-17 o 09 08 31" src="https://user-images.githubusercontent.com/20516055/185056618-2e500e5f-4236-4fa8-99c3-97b86c77f24c.png">

Reply:
It looks like this with the default padding:
<img width="331" alt="image" src="https://user-images.githubusercontent.com/10947344/185305975-416380f0-032e-44e4-b68f-fccd435b7eed.png">
I just felt like it was to much and that it makes the header feel disconnected from the content